secrets/hashivault/example_test.go

```diff
@@ -24,11 +24,9 @@ import (
 )
 
 func ExampleOpenKeeper() {
-	// This example is used in https://gocloud.dev/howto/secrets/#vault-ctor
-
-	// import _ "gocloud.dev/secrets/hashivault"
-
-	// Variables set up elsewhere:
+	// PRAGMA(gocloud.dev): Package this example for gocloud.dev.
+	// PRAGMA(gocloud.dev): Add a blank import: _ "gocloud.dev/secrets/hashivault"
```

Comment:
Don't think you need this since this is not a URL example.

Reply:
Fixed.